src/components/VFRChartsList.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+'use strict';
+
+import React, { Component, PropTypes } from 'react';
+import {
+  Dimensions,
+  ListView,
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+import VFRChart from '../model/VFRChart';
+import Scenes from '../containers/Scenes';
+import Colors from '../styles/Colors';
+import FontStyles from '../styles/FontStyles';
+
+class VFRChartsList extends Component {
```

Comment:
@jrvanderveen this is the basic list view skeleton I was using to test scrolling. You will have to change the component that is being rendered in `renderRow` and do whatever else you see fit.